### PR TITLE
Fix demos

### LINKED ISSUE #152: Fix demos

### PR DESCRIPTION
**Pull request by sisou**
Resolves #152.

Some demos are not salvagable and I did not add new ones. But at least it's something for additional testing.

### CODE REVIEW

File: demos/ExportWords.html [hunk 1]
```diff
@@ -12,71 +12,78 @@
...
 
 <script>
     (async () => {
-        // Because the mocked KeyStore.prototype.get function below is defined in this
-        // window's scope, Nimiq also needs to be defined and initialized here
-        await Nimiq.WasmHelper.doImportBrowser();
-        Nimiq.GenesisConfig.test();
+        // await Nimiq.WasmHelper.doImportBrowser();
+        // Nimiq.GenesisConfig.test();
     })();
```
Comment (Bettelstab) [inline demos/ExportWords.html]: Remove It?

File: demos/ExportWords.html [hunk 2]
```diff
@@ -12,71 +12,78 @@
...
 
     document.querySelector('.bip39').addEventListener('click', async () => {
 
-        const keyguard = window.open('../src/request/export-words/');
+        const keyguard = window.open('../src/request/export/', 'CreateDemo',
```
Comment (Bettelstab) [inline demos/ExportWords.html]: 'Export Demo' or 'Demo'

File: demos/Import.html [hunk 3]
```diff
@@ -9,24 +9,28 @@
...
 
 <script>
-    document.querySelector('#import-words').addEventListener('click', () => requestImport('words'));
-    document.querySelector('#import-file').addEventListener('click', () => requestImport('file'));
+    document.querySelector('#import').addEventListener('click', () => requestImport());
 
-    async function requestImport(type) {
-        const keyguard = window.open(`../src/request/import-${type}/`, 'keyguard');
+    async function requestImport() {
+        const keyguard = window.open(`../src/request/import/`, 'CreateDemo',
```
Comment (Bettelstab) [inline demos/Import.html]: Not CreateDemo

File: demos/RemoveKey.html [hunk 4]
```diff
@@ -12,36 +12,35 @@
...
-        // Because the mocked KeyStore.prototype.get function below is defined in this
-        // window's scope, Nimiq also needs to be defined and initialized here
         await Nimiq.WasmHelper.doImportBrowser();
-        Nimiq.GenesisConfig.test();
+        // Nimiq.GenesisConfig.test();
     })();
 
     document.querySelector('button').addEventListener('click', async () => {
-        const keyguard = window.open('../src/request/remove-key/');
+        const keyguard = window.open('../src/request/remove-key/', 'CreateDemo',
```
Comment (Bettelstab) [inline demos/RemoveKey.html]: Demo

File: demos/SignTransaction.html [hunk 5]
```diff
@@ -51,34 +54,34 @@
     (async () => {
         await Nimiq.WasmHelper.doImportBrowser();
```
Comment (Bettelstab) [inline demos/SignTransaction.html]: doImport?

File: demos/ExportWords.html (hunk 1 above)
Comment (sisou) [reply demos/ExportWords.html]: Fixed it instead ;)

**Comment by sisou** [comment]
Rebased and fixed export demo.

File: demos/Export.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+<body>
+<link href="../src/nimiq-style.css" rel="stylesheet">
+<script src="../node_modules/@nimiq/core-web/web-offline.js"></script>
+<script src="../src/lib/Constants.js"></script>
+<script src="../src/lib/Key.js"></script>
+<script src="../src/lib/KeyStore.js"></script>
+<script src="../node_modules/@nimiq/rpc/dist/rpc.umd.js"></script>
+
+<div>
+    <button class="bip39">Export Words (BIP39)</button>
```
Comment (nibhar) [inline demos/Export.html]:
```suggestion
    <button class="bip39">Export (BIP39)</button>
```
following lines the same.

**Pull request merged**